Deploy-CAM.ps1

```diff
@@ -2261,19 +2267,43 @@ function Deploy-CAM() {
     }
 }
 
-function Confirm-ModuleVersion {
+function Confirm-ModuleVersion()
```

Comment:
Confirm-ModuleVersion was giving me errors on my laptop. The reason was or seems to be that AzureRM as a module is not installed with the newest SDK installer, just a new version of Azure module (and all the AzureRM sub-modules...) so this seems to be a valid configuration...